lib/ansible/modules/network/aci/aci_firmware_policy.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: aci_firmware_policy
+
+short_description: This creates a firmware policy
```

Comment:
In the module index it doesn't look good if it starts with 'This creates'.
Simply use `Manage firmware policies`.

Reply:
The DOCUMENTATION string is very indented, I would prefer we use the same format as the other ACI modules. Only indent when necessary (2 spaces).